### DISCUSSION TITLE
TTS - volume problem.

### DISCUSSION THREAD

**Discussion by senderbell** [music-assistant/support#2051] [Q&A] [answered]
Hi.

Does anyone still have an issue with TTS?
When the music is not playing and the speaker is turned off, starting TTS will turn on the speaker and set it to the last known volume.
Then TTS increases the volume by 10%, plays the message, decreases the volume to 0, and turns off the speaker.
The next TTS, after increasing the volume by 10%, plays the message very quietly. (volume = 10)
Is there a way to prevent TTS from reducing the volume to 0 after playing the message?

This issue only occurs when playing TTS on the Music Assistant entity.

The second issue is that the TTS triggered on the Music Assistant entity is about 2 times quieter than the same TTS triggered on the google cast entity with the same volume setting.

**Comment by marcelveldt** [comment]
Are you using the native cast players in MA?

**Comment by senderbell** [comment]
I use a Google speaker. Everything was working fine until the last update in HACS. Yesterday I updated HA to the latest version and the issue with reducing the volume to 0 disappeared, but TTS is still playing very quietly. Much quieter than the music being played.

**Comment by senderbell** [comment]
OK. I found the solution. I had to enable 'Prepend notification sound to TTS announcements' and now TTS plays loudly and clearly.

**Reply by OzGav** [reply, marked as answer]
Yeah it's not s solution its a bug that we do know about. Hopefully we'll be fixed in the near future
